lib/circleci.js

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable camelcase */
+/* eslint-disable no-console */
+'use strict';
```

Comment:
@makarandp0 Nitpick:
```suggestion
'use strict';

```